plugins/parsers/csv/README.md

```diff
@@ -33,8 +33,35 @@ values.
   ## If this is not specified, type conversion will be done on the types above.
   csv_column_types = []
 
-  ## Indicates the number of rows to skip before looking for header information.
+  ## Indicates the number of rows to skip before looking for metadata and header information.
   csv_skip_rows = 0
+  
+  
+  ## Indicates the number of rows to parse as metadata before looking for header information. 
+  ## By default, the parser assumes there are no metadata rows to parse. 
+  ## If set, the parser would use the provided separators in the cvs_metadata_separators to look for metadata.
+  csv_metadata_rows = 0
+  
+  ## A list of metadata separators. If csv_metadata_rows is set,
+  ## cvs_metadata_separators must contain at least one separator.
+  ## Please note that separators are case sensitive.
+  ## For example:
+  ## cvs_metadata_separators = [",", ":", "="]
+  ## to parse "Key1,Value1" "Key2:Value2" "Key3=Value3"
+  ## the returned (key,value) pares would be as follows:
+  ## (Key1,Value1), (Key2,Value2), (Key3,Value3)
+  cvs_metadata_separators = [","]
```

Comment:
Would `=` or `:` not be a better default value? It seems to me those are more commonly used to express key and values.

Reply:
The reason I have used ',' is because its a CSV file and logically you would separate the key value with commas.

For what I need this feature, I need ':' and '='. But if I had the option to rewrite the code I would not use two different separator characters.

Reply:
In CSV, you separate the columns/values with `,` There are no key-value pairs. I would still use either `=` or `:` as default example as that will be more clear to users what it does.